package.json

```diff
@@ -10,7 +10,9 @@
     "lint": "eslint --report-unused-disable-directives --ignore-path .gitignore .",
     "fix": "npm run lint -- --fix",
     "mocha": "mocha",
-    "test": "mocha"
+    "test": "npm run test:node-sass && npm run test:dart-sass",
+    "test:node-sass": "mocha",
+    "test:dart-sass": "cross-env SASS_COMPILER=sass mocha"
```

Comment:
Out of curiosity could you do something like `mocha -- sass` and pull the compiler off of `argv` instead of introducing cross-env?

Reply:
It could probably work (not `mocha -- sass` but `mocha -- --sass`, but seems more hacky from a quick look. Feel free to tweak it, though.

Reply:
Actually, wait. Let me try something.

Reply:
I pushed a patch which does work but still seems hacky, but correct me if I'm wrong. Aren't we abusing the fact that mocha doesn't throw on unknown command line options?

Reply:
I'm fine with it until it's a problem :)